ubuntu/packages

```diff
@@ -1,48 +1,61 @@
-cmake
```

Comment:
Sure, a deb will help. However, we might easily let people decide on their own build tools instead of shipping them ourselves. I am thinking of doing similarly to the Arch one as well.

Reply:
I think that the loss of functionality for not having cmake and make is bigger than what's gained in space by not installing them. CMake is less than 30M and make is only ~500K. ROOT alone is about 900M. It's much better to include them in the image. We could add variants based on tags like `minimal` and `full`, though. Then we can have `Dockerfile.${tag}` for each variant and images suited for different use cases.